enterprisemanagementv1/enterprise_management_v1_integration_test.go

```diff
@@ -19,780 +19,405 @@
-			Line_item_id            int            `json:"line_item_id"`
-		}
-
-		accountPayload := Payload{
-			Type:  "SUBSCRIPTION",
-			State: "ACTIVE",
-			Payment_methods: Payment_method{
-				Start_date: "2020-03-01T07:00:00.000Z",
-				End_date:   "2020-11-30T08:00:00.000Z",
-			},
-			Subscription_amount:     100,
-			Quantity:                10,
-			Billing_frequency:       "M",
-			Charge_agreement_number: "0099342614",
-			Partner_customer_number: "0003615466",
-			Configuration_id:        "5900A5D20190517",
-			Part_number:             "D019JZX",
-			Order_id_number:         "150418156",
-			Sales_doc_type_code:     "",
-			Renewal_mode_code:       "T",
-			Renewal_date:            "",
-			Terminate_renewal:       false,
-			Line_item_id:            10,
-		}
-
-		accountPayloadJson, _ := json.Marshal(accountPayload)
-
-		req, err := http.NewRequest("PATCH", url, bytes.NewBuffer(accountPayloadJson))
-		Expect(err).To(BeNil())
-
-		err = amAuth.Authenticate(req)
-		Expect(err).To(BeNil())
-
-		// Send req using http Client
-		client := &http.Client{}
-		resp, err := client.Do(req)
-		Expect(err).To(BeNil())
-
-		buf, err := ioutil.ReadAll(resp.Body)
-		Expect(err).To(BeNil())
-		Expect(buf).ToNot(BeNil())
+			secondExampleAccountGroupId = createSecondExampleAccountGroupResponse.AccountGroupID
+		})
 	})
 
-	It("Successfully create enterprise", func() {
-		shouldSkipTest()
+	Describe(`ListAccountGroups - List account groups`, func() {
+		BeforeEach(func() {
+			shouldSkipTest()
+		})
+		It(`ListAccountGroups(listAccountGroupsOptions *ListAccountGroupsOptions)`, func() {
 
-		options := service.NewCreateEnterpriseOptions(account_id, "IBM", owner_iam_id)
+			listAccountGroupsOptions := &enterprisemanagementv1.ListAccountGroupsOptions{
+				EnterpriseID: &enterpriseId,
+			}
 
-		result, detailedResponse, err := service.CreateEnterprise(options)
-		Expect(err).To(BeNil())
-		Expect(detailedResponse.StatusCode).To(Equal(202))
+			listAccountGroupsResponse, response, err := enterpriseManagementService.ListAccountGroups(listAccountGroupsOptions)
 
-		enterprise_id = *result.EnterpriseID
-		enterprise_account_id = *result.EnterpriseAccountID
-		Expect(err).To(BeNil())
+			Expect(err).To(BeNil())
+			Expect(response.StatusCode).To(Equal(200))
+			Expect(listAccountGroupsResponse).ToNot(BeNil())
 
+		})
 	})
 
-	It("Successfully get account", func() {
-		shouldSkipTest()
-
-		apiUrl := testConfig["AM_HOST"]
-		resource := "/coe/v2/accounts/" + account_id
-		u, err := url.ParseRequestURI(apiUrl)
-		Expect(err).To(BeNil())
-
-		u.Path = resource
-		urlStr := u.String()
-
-		url := urlStr
-
-		req, err := http.NewRequest("GET", url, nil)
-		Expect(err).To(BeNil())
-
-		err = amAuth.Authenticate(req)
-		Expect(err).To(BeNil())
+	Describe(`ListAccountGroups - List account groups with paging`, func() {
+		BeforeEach(func() {
+			shouldSkipTest()
+		})
+		It(`ListAccountGroups(listAccountGroupsOptions *ListAccountGroupsOptions)`, func() {
```

Comment:
Let's re-code this test so that it uses a loop to retrieve a page at a time (perhaps use limit=1?) and accumulate the results in a slice, similar to how the user-management int tests exercise the list-type operations.